pkg/ui/src/views/databases/containers/databaseSummary/index.tsx

```diff
@@ -42,6 +42,7 @@ interface DatabaseSummaryActions {
   refreshDatabaseDetails: typeof refreshDatabaseDetails;
   refreshTableDetails: typeof refreshTableDetails;
   refreshTableStats: typeof refreshTableStats;
+  updateOnLoad: boolean;
```

Comment:
`updateOnLoad` prop has to be set `true` by default. Otherwise it's undefined and doesn't allow to load data in places which have to preserve old logic.

Reply:
@koorosh where can I set a default value for this? the definition is in an interface. Also the only other usage is on the Grants page which does not do a per-item load.

Reply:
it can be defined as static field for React class component:
```typescript
export class DatabaseSummaryBase extends React.Component<DatabaseSummaryProps, DatabaseSummaryState> {
  static defaultProps: Partial<DatabaseSummaryProps> = {
    updateOnLoad: true
  }
  ...
}
```